presto-postgresql/src/test/java/com/facebook/presto/plugin/postgresql/TestPostgreSqlDistributedQueries.java

```diff
@@ -78,6 +79,24 @@ public void testViews()
         execute("DROP VIEW IF EXISTS tpch.test_view");
     }
 
+    @Test
+    public void testSelectNativeDataTypeColumn()
+            throws Exception
+    {
```

Comment:
Please add preconditions check to ensure `test_other_data_type` table does not exist before you attempt creating it.